diff --git a/dist/chunk-VQG3VKOR.js b/dist/chunk-VQG3VKOR.js
index 34a3d6edd493e8866920e5e6e2c4c4ae759dba23..a518f77ec5752d112ba707e3a885b6c011a1b870 100644
--- a/dist/chunk-VQG3VKOR.js
+++ b/dist/chunk-VQG3VKOR.js
@@ -698,13 +698,19 @@ addEffects_fn = function() {
       }
     );
 };
-var client;
+var root = (typeof self === 'object' && self.self === self && self) ||
+  (typeof global === 'object' && global.global === global && global) ||
+  this
+
+var client = root.wagmiClient;
 function createClient(config) {
   const client_ = new Client(config);
+  root.wagmiClient = client_;
   client = client_;
   return client_;
 }
 function getClient() {
+  client = client || root.wagmiClient;
   if (!client) {
     throw new Error(
       "No wagmi client found. Ensure you have set up a client: https://wagmi.sh/react/client"